### PR TITLE
Mobile version body

### PR DESCRIPTION
**Pull request by ijeomaodinko**
This a project on portfolio body, it shows the following:
The content of portfolio.
The index.html file and style.css file. 
It contains the linters.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        <p>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required.</p>
+        <ul class="list3">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See project</button>
+        </div>
+    </section>
```
Comment (BenMukebo) [inline index.html:129]: - [ ] Great work, your design looks awesome, however, I think there are a few things you will need to implement before you to the next project; please wrap all cards above in one main section tag and apply `display: grid` to it to align the cards inside.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,185 @@
...
+          <li class="special">Back End Dev</li>
+          <li class="special">2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; no 
+          accounts or sign-ups required.</p>
+        <ul class="list2">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
```
Comment (BenMukebo) [inline index.html:106]: - [ ] Take a look at this image below, kindly match your design with the Figma one
width

- It should be like this instead ⏬ 
oriwd

File: index.html [hunk 3]
```diff
@@ -0,0 +1,185 @@
...
+          <li class="special">2015</li>
+        </ul>
+        <p>A daily selection of privately personalized reads; 
+          no accounts or sign-ups required.</p>
+        <ul class="list3">
+          <li>html</li>
+          <li>css</li>
+          <li>javascript</li>
+        </ul>
+        <button class="btn">See project</button>
```
Comment (BenMukebo) [inline index.html:127]: - [ ] As required, please add button actions for all you `see project and get my resume` button as shown in this image below ⏬ 
action

**Review by BenMukebo — CHANGES_REQUESTED** [review]
### Hi *** @ijeomaese, ***

You did a great job so far 👏🏼 👏🏼
There are some issues that you still need to work on to go to the next project but you are almost there!



### To highlight 💪🏻

- No linter errors. ✔️ 
- Good commit messages ✔️ 
- Professional README ✔️ 
 👍  Good job!


## STATUS: CHANGES REQUESTED  ♻️ 
_Please Check the comments under the review and make all required changes_




## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏 💻  

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me on( @BenMukebo ) in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 4]
```diff
@@ -0,0 +1,352 @@
...
+
+.SectionText {
+  padding: 3rem 0 3rem 3rem;
+}
+
+h2 {
+  color: #172b4d;
+  font-size: 35px;
+  margin: -2rem 0 1rem 0;
+}
```
Comment (phelian23) [inline style.css:223]: - [ ] [Optional]

Kindly use CSS classes for styling the HTML elements As CSS classes are reusable. 👍🏻

File: style.css [hunk 5]
```diff
@@ -0,0 +1,352 @@
...
+  align-items: center;
+  background-color: #f7f7f9;
+  border-radius: 15px;
+  margin-top: 3rem;
+  padding: 0.5rem 0;
+}
+
+h3 {
+  padding-left: 2rem;
+}
```
Comment (phelian23) [inline style.css:288]: - [ ] [Optional]
Kindly use CSS classes for styling the HTML elements As CSS classes are reusable. 👍🏻

File: style.css [hunk 6]
```diff
@@ -0,0 +1,352 @@
...
+  padding: 6rem 0 3rem 3rem;
+}
+
+.sectiondiv {
+  background-color: white;
+  background-size: 100%;
+  border: 2px solid white;
+  border-radius: 0 50px 0 0;
+  margin-top: 4rem;
+}
```
Comment (phelian23) [inline style.css:251]: - [ ] [Optional]
Kindly make sure that your design match the Figma template design as much as possible (e.g. font, colors, images, tests, margins)

Your design
Capture

Figma temp
figma
late

**Review by phelian23 — APPROVED** [review]
Hi @ijeomaese, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

### Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**